dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/services/impl/MemoryPersistenceManagerImpl.java

```diff
@@ -1384,4 +1385,38 @@ public String getPersistenceType() {
         return "MEM";
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public List<String> getGroupNamesForJobID(long jobInstanceID) throws NoSuchJobInstanceException {
+        // TODO Auto-generated method stub
+        return null;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public boolean isJobIDAccessibleByAnyGroupName(long jobInstanceID, List<String> subjectGroups) throws NoSuchJobInstanceException {
+        // TODO Auto-generated method stub
+        return false;
+    }
+
+    /** {@inheritDoc} */
+//    @Override
+//    public Collection<? extends WSJobInstance> getJobInstancesWithGroupSecurity(WSSearchObject wsso, int page, int pageSize) {
+    // TODO Auto-generated method stub
+//        return null;
+//    }
+
+    /** {@inheritDoc} */
+    @Override
+    public List<JobInstanceEntity> getJobInstancesForSubjectGroupNames(List<String> listOfGroupsForSubject, String subject) {
+        // TODO Auto-generated method stub
+        return null;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public JobInstanceEntity updateJobInstanceWithGroupNames(long jobInstanceId, Set<String> groupNames) {
+        return null;
+    }
+
 }
```

Comment:
It seems like the write/update should throw UnsupportedOperationException while the read should maybe return an empty list.  (First obsolete methods should be removed).